src/Avatar.svelte

```diff
@@ -0,0 +1,50 @@
+<script lang="typescript">
+  import { onMount } from 'svelte';
+  import { createIcon } from '@app/blockies';
+  import { isAddress } from './utils';
+  
+  export let source: string;
+  export let icon: boolean = true;
```

Comment:
What does this do?

Reply:
To be able to define which style of avatar the component should show I introduced this icon property that reduces the avatar to small icon size by default.
If one doesn't want a icon size avatar one should pass `icon={false}`.

Eventually there is a way to make this happen with CSS without hardcoding it, else we could also do a `style` prop with some options like `icon`, `header`, etc`.

Reply:
For example if using an avatar as inline icon we want to have `margin-right: 0.5rem`, but as part of the users or orgs header it shouldn't have margins.

And the blockies div, since they are not images but divs with a background image they behave very differently to images, regarding trying to fill their width and height as part of an inline element..
If we give them as inline icons a width and height of 1rem they behave as expected..
When they form part of a fixed size div it's not needed to define their size since they adapt to the parent div.

Reply:
Hmmm can we call this `inline` then and have it default to false? I think it's nicer if components have params that default to `false`, as the default behavior can change with just `myParam` instead of `myParam={false}`.

Reply:
Yes that should be no problem 👍

Reply:
ea01cf541e06988ff6a41785e9076616531958d0 fixes this